### PR TITLE
Adds instructions for building iOS from Windows

### PR DESCRIPTION
**Pull request by emaf**
iOS workloads need to be updated to include the version with Windows support.

### CODE REVIEW

**Review by rolfbjarne — APPROVED** [review]

**Review by jonathanpeppers — COMMENTED** [review]
Can you link to a commit that has the builds I'd need? I'd like to try these out, thanks.

**Comment by emaf** [comment]
@jonathanpeppers this is the PR https://github.com/xamarin/xamarin-macios/pull/10638, the build pipeline is still running but the packages were already produced

**Review by jonathanpeppers — CHANGES_REQUESTED** [review]
Thanks, I'll try these out: https://github.com/xamarin/xamarin-macios/pull/10638

Since we are likely shipping the packages on the current `README.md` for a Preview 1, let's wait to merge this for a few days.

We should probably update download links at the same time as adding these instructions, though.

**Review by Redth — APPROVED** [review]

**Comment by rolfbjarne** [comment]
> @rolfbjarne I just updated the links to the iOS workloads, please check if those are correct. Thanks!

I think you have to update here too:

https://github.com/dotnet/net6-mobile-samples/blob/0890331cbc5e41b576f1216a3d5a57634bf170af/azure-pipelines.yml#L14-L15

File: README.md [hunk 1]
```diff
@@ -16,8 +16,12 @@ Android:
 
 iOS:
 
-* Windows: [Microsoft.NET.Workload.iOS.14.3.100-ci.main.1079.msi](https://bosstoragemirror.azureedge.net/wrench/main/f01fde5cd9a7ffffcdc8d241200c35988700fa00/4449408/package/Microsoft.NET.Workload.iOS.14.3.100-ci.main.1079.msi)
-* macOS: [Microsoft.iOS.Bundle.14.3.100-ci.main.1079.pkg](https://bosstoragemirror.azureedge.net/wrench/main/f01fde5cd9a7ffffcdc8d241200c35988700fa00/4449408/package/notarized/Microsoft.iOS.Bundle.14.3.100-ci.main.1079.pkg)
+These builds are newer than [.NET 6 Preview 1][net6preview1] and
+require Xcode 12.4. Use downloads from [.NET 6 Preview
+1][net6preview1] for Xcode 12.3:
```
Comment (jonathanpeppers) [inline README.md:21]: @Redth I put this note for now. Do we want to wait a little while to merge this?

This literally _just_ came out: https://devblogs.microsoft.com/dotnet/announcing-net-6-preview-1/

**Review by jonathanpeppers — APPROVED** [review]

**Pull request merged**